src/config/services/managing.md

```diff
@@ -102,6 +102,7 @@ See [runsvdir(8)](https://man.voidlinux.org/runsvdir.8) and
 
 ### Booting a different runsvdir
 
-To boot a different runsvdir, the name of the runsvdir can be added to the
-[kernel command-line](../kernel.html#cmdline). As example adding `single` to the
-[kernel command line](../kernel.html#cmdline) will boot the `single` runsvdir.
+To boot a `runsvdir` other than `default`, the name of the desired `runsvdir`
```

Comment:
"runsvdir" is not in backticks earlier in the section, which i think makes sense, so maybe remove the backticks here, and in the next instance?

Reply:
This was mostly because the other page constantly used backticks for runsvdir. It probably makes more sense to not use backticks unless we are talking about the command, though. Will change!